setup.py

```diff
@@ -67,16 +77,12 @@
 install_requires = [
     'bravado>=9.0.6,<10.2',
     'click>=6.7,<6.8',
-    'cwltool==1.0.20180326152342',
-    'pyOpenSSL==17.3.0',  # FIXME remove once yadage-schemas solves deps.
     'reana-commons>=0.3.1,<0.4',
-    'rfc3987==1.3.7',  # FIXME remove once yadage-schemas solves deps.
-    'strict-rfc3339==0.7',  # FIXME remove once yadage-schemas solves deps.
     'tablib>=0.12.1,<0.13',
-    'webcolors==1.7',  # FIXME remove once yadage-schemas solves deps.
-    'yadage-schemas==0.7.16',
 ]
 
+# install_requires += extras_require['yadage'] + extras_require['cwl']
```

Comment:
Should probably uncomment it for now to support all the workflow engines we have... Otherwise we would have to loudly specify `pip install reana-client[yadage,cwl]` everywhere

Reply:
Shall we uncomment this and rebase? Seems to be ready